install_sct.bat

```diff
@@ -0,0 +1,45 @@
+:: Usage: install_sct.bat <version>
+:: e.g.
+::        install_sct.bat 5.5
+
+:: Fetch version number (should be git branch, git tag, etc.)
+set git_ref=%1

```

Comment:
This usage could maybe be tweaked a little. e.g. Hardcoding the version to install for each release?

The main reason I did it this way was to allow for testing PRs, etc.